pkgs/top-level/all-packages.nix

```diff
@@ -9961,6 +9961,8 @@ with pkgs;
 
   lolcat = callPackage ../tools/misc/lolcat { };
 
+  lolcat-rs = callPackage ../tools/misc/lolcat-rs { };
```

Comment:
Use the new [`pkgs/by-name`](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name) format so you don't have to edit this file